automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/PageUtil.scala

```diff
@@ -31,8 +31,9 @@ trait PageUtil[P <: Page] extends FireCloudView with WebDriverIdLogging { self:
     }
     Try (awaitReady()) match {
       case Success(_) =>
-      case Failure(_) =>
+      case Failure(f) =>
         log.error(s"Timed out ($defaultTimeOutInSeconds seconds) waiting for page ${this.url}")
+        throw(f)
```

Comment:
this is unrelated to the sign-in component, but I noticed in the failure case, we logged but continued on. I think we should fail here.

Reply:
shall we put `f` in the log message too?